airflow/models/baseoperator.py

```diff
@@ -1188,6 +1196,21 @@ def dag(self, dag: DAG | None):
 
         self._dag = dag
 
+    @property
+    def task_display_name(self) -> str:
+        return self._task_display_property_value or self.task_id
+
+    @property
+    def label(self) -> str | None:
```

Comment:
We can do this on AbstractOperator instead of duplicating the implementation on two classes. We need to add an abstract property on AbstractOperator to access `task_display_name` but I think otherwise it’d work?

Reply:
Would it be ok if we do this in a separate PR?

Reply:
Yeah probably, but since there’s another issue to fix (the `dump_only` one above) it might be better to just do it here…

Reply:
Adjusted.